t/t3405-rebase-malformed.sh

```diff
@@ -69,4 +69,11 @@ test_expect_success 'rebase commit with diff in message' '
 	test_cmp G G0
 '
 
+test_expect_success 'rebase exec modifies rebase-todo' '
+        git rebase HEAD -x "echo x \"echo foo >H\" >> \"$(
```

Comment:
That's testing a very different thing, though.  I've moved the test into a new file: t3428-rebase-edit-todo.sh, since it doesn't really fit into the others.